src/main/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2RequestChangesNotification.java

```diff
@@ -46,9 +48,11 @@ public void sendToApplicant1(CaseData caseData, Long id) {
         if (caseData.getDivorceOrDissolution().isDivorce()) {
             templateVars.put(APPLICATION.toLowerCase(Locale.ROOT), FOR_DIVORCE);
             templateVars.put(END_JOINT_APPLICATION_LINK, configTemplateVars.get(SIGN_IN_DIVORCE_URL));
```

Comment:
we don't need that END_JOINT_APPLICATION_LINK or APPLICATION variable anymore I don't think

Reply:
Yeah good point! will remove it